src/components/Dialogs/FixedDimension.js

```diff
@@ -3,13 +3,28 @@ import {
     ouIdHelper,
     DIMENSION_ID_ORGUNIT,
 } from '@dhis2/analytics'
+import i18n from '@dhis2/d2-i18n'
+import { Checkbox } from '@dhis2/ui'
 import PropTypes from 'prop-types'
 import React from 'react'
 import { connect } from 'react-redux'
 import { tSetCurrentFromUi } from '../../actions/current.js'
 import { acAddMetadata } from '../../actions/metadata.js'
 import { acSetUiItems, acAddParentGraphMap } from '../../actions/ui.js'
+import {
+    DIMENSION_TYPE_EVENT_STATUS,
+    DIMENSION_TYPE_PROGRAM_STATUS,
+} from '../../modules/dimensionTypes.js'
 import { removeLastPathSegment, getOuPath } from '../../modules/orgUnit.js'
+import {
```

Comment:
Wondering if this the best module for these constants or if another location would be more appropriate. If it is only used by FixedDimension, perhaps it can just be internal to that component?

Reply:
One constant is used in `modules/visualization`.
I'm not against moving them, they are used for both program and event stage.